WordPress/WordPressIntents/en.lproj/Sites.strings

```diff
@@ -0,0 +1,12 @@
+"BOl9KQ" = "There are ${count} options matching ‘${site}’.";
+
+"ILcGmf" = "Site";
+
+"cyajMn" = "Site";
+
+"gpCwrM" = "Select Site";
+
+"s4dJhx" = "Just to confirm, you wanted ‘${site}’?";
+
+"tVvJ9c" = "Select Site Intent";
```

Comment:
There are no way to have nicer, better keys? (I'm assuming those are Xcode-generated, like when you as Xcode to extract the strings from a XIB or Storyboard, but for Intents?)

Not even sure those keys are even picked up by our localization tools currently, but when we upgrade our tools so that they do, it would be nice to rely on keys with clear intent and naming (reverse-DNS style for example, like we did for Widgets)

Reply:
Yes – definitely agree with all of this – for now this was to silence warnings in the project and get the translations moving. Worst-case, we might end up just hand-copying them into position until the tooling improvements are present.